src/api/app/mailers/event_mailer.rb

```diff
@@ -61,11 +57,17 @@ def set_event_headers
     headers(@event.custom_headers)
   end
 
-  def set_sender
+  def sender_realname
     return unless @event
 
-    @sender = @event.originator
-    @sender ||= DefaultSender.new(@configuration.admin_email, 'OBS Notification') # rubocop:disable Naming/MemoizedInstanceVariableName
+    if @event.originator
+      realname = @event.originator.realname
+      realname += " (#{@event.originator.login})"
+    else
+      realname = 'OBS Notification'
+    end
+
+    realname
```

Comment:
```suggestion
  return 'OBS Notification' unless @event.originator
  
  "#{@event.originator.realname} (#{@event.originator.login})"
```

I would use a guard clause instead of the conditional. Less code, and more clear. What do you think?